### DISCUSSION TITLE
LaunchRotation/LaunchRotationRadians Broken

### DISCUSSION THREAD

**Discussion by SydWad** [FakeFishGames/Barotrauma#15430] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

XML error.

It appears LaunchRotation and LaunchRotationRadians no longer works on projectiles launched by turrets.
![image](https://github.com/user-attachments/assets/05a94251-e960-43bb-9626-8c5872f97c14)

### Reproduction steps

Here is an example of the code i was using.
In my case i am attempting to make a new type of Boarding Pod depth charge ammo.
`<Projectile launchrotation="90" characterusable="false" launchimpulse="20" damagedoors="true" StickToCharacters="true" StickToStructures="True" StickToDeflective="true" LightTargetMassThreshold="1500" >`

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.7.7.0 (Winter Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by SydWad** [comment]
Additional problem. It appears suitableProjectiles isnt supported by turrets for some reason.

**Comment by Regalis11** [comment]
The issue has been addressed in our private repo now, fix coming up in the next unstable build and the upcoming update! :)
